tests/autoyast/installation.pm

```diff
@@ -22,7 +22,6 @@ use testapi;
 use utils;
 
 my $confirmed_licenses = 0;
-my $stage;
```

Comment:
why we need it ? I don't see any functionality loss. 
before we had  :
installation-y2logs-stage1_error1.tar.bz2
installation-y2logs-stage1_error2.tar.bz2
installation-y2logs-stage2_error3.tar.bz2

and after my commit we will have :
installation-y2logs-error1.tar.bz2
installation-y2logs-error2.tar.bz2
installation-y2logs-error3.tar.bz2

anyway to understand what "stage1" / "stage2" suppose to mean you need to read the code. And if you read my code in a same you will be able to understand meaning of error1 / error2 / error3